docs/components/COMPONENT_REPLACEMENT.md

```diff
@@ -1,22 +1,21 @@
 # Component replacement
 
-## Overview
-
+The goal of the feature is to make a component replaceable at runtime.
 This replacement mechanism integrated with application customization relies on [Angular component factories](https://angular.io/guide/dynamic-component-loader)
-in order to create the right component at runtime.
+in order to create the component at runtime.
 You can see a lot of details in the link above, and you'll probably notice that it is very verbose and cumbersome to
 implement:
 
 * Inputs and outputs have to be bound again, outside the template
 * Change detection has to be adapted in order to pass potential changes of those inputs when the parent component changes
 
-In order to alleviate that, we provide a directive ``c11n`` that, when applied to an ``ng-template`` and given the following information will handle all the wiring behind the scene:
+In order to alleviate that, we provide a directive ``c11n`` that, when applied to an ``ng-template`` and given the following information, will handle all the wiring behind the scene:
 
 * The type of the component to instantiate
 * The inputs and outputs to bind
 * The potential configuration override to apply
 
-Now let's see how to use that mechanism in practice.
+Now let's see how to set this mechanism up to create your replacable components.
 
 ## Prepare your base app to register custom components
 
```

Comment:
I don't think we should keep this part because it refers to an architecture and script that is no longer available in this repository (true up to line 104). Or at least, it should be carefully reviewed so it stays compatible with everything available here